packages/opentelemetry-host-metrics/package.json

```diff
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "@opentelemetry/sdk-metrics": "^1.8.0",
-    "systeminformation": "^5.21.20"
+    "systeminformation": "5.22.9"
   },
   "homepage": "https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/opentelemetry-host-metrics#readme"
-}
+}
```

Comment:
Please add back the EOF.

Reply:
Interesting, you would think that linting would pick this up

Reply:
We currently have linting of .ts and .md files, but not of the package.json files.